packages/synapse-react-client/stories/IconSvg.stories.tsx

```diff
@@ -8,7 +8,7 @@ const meta = {
   render: args => {
     return (
       <div style={{ display: 'flex', flexWrap: 'wrap', flexDirection: 'row' }}>
-        {IconStrings.sort().map(icon => (
+        {IconStrings.toSorted().map(icon => (
```

Comment:
To sorted actually broke Chromatic...my bad
```suggestion
        {IconStrings.sort().map(icon => (
```